### ISSUE TITLE
[Model]:  Can you provide the trained model?

### ISSUE THREAD

**Issue by mynewstart** [hpcaitech/ColossalAI#3348]
### Describe the feature

My machine can't run the training stage, can you provide model so I can use it to inference.

**Comment by djaffer** [comment]
Possibly on Huggingface we can use for inference.

**Comment by mynewstart** [comment]
Do you have model name?

**Comment by FrankLeeeee** [comment]
Hi, we currently don't provide the trained model due to the license restriction of the LLaMA model.
https://github.com/facebookresearch/llama/blob/main/LICENSE

One possible way to release the model is to release its LORA weights, we have this on our plan, possibly release next week.

**Comment by mynewstart** [comment]
Got it, thx~

**Comment by djaffer** [comment]
Yeah if training is required then I wonder how immediate useful it is. There are many other models out there that too requires training.

**Comment by timatom** [comment]
You'd be surprised how many are getting pretty far with cheaply training their own models:

https://github.com/lm-sys/FastChat

https://vicuna.lmsys.org/

**Comment by djaffer** [comment]
Thanks for sharing. Another one is https://github.com/nomic-ai/gpt4all. 
The cost is too prohibitive to run some of these on consumer-grade hardware.
